authority/provisioner/sign_ssh_options_test.go

```diff
@@ -384,3 +440,120 @@ func Test_sshValidityModifier(t *testing.T) {
 		})
 	}
 }
+
+func Test_sshModifierFunc_Modify(t *testing.T) {
+	type args struct {
+		cert *ssh.Certificate
+	}
+	tests := []struct {
+		name    string
+		f       sshModifierFunc
+		args    args
+		wantErr bool
+	}{
+		{"ok", func(cert *ssh.Certificate) error { return nil }, args{&ssh.Certificate{}}, false},
+		{"fail", func(cert *ssh.Certificate) error { return fmt.Errorf("an error") }, args{&ssh.Certificate{}}, true},
+	}
+	for _, tt := range tests {
+		t.Run(tt.name, func(t *testing.T) {
+			if err := tt.f.Modify(tt.args.cert); (err != nil) != tt.wantErr {
+				t.Errorf("sshModifierFunc.Modify() error = %v, wantErr %v", err, tt.wantErr)
+			}
+		})
+	}
+}
+
+func Test_sshDefaultDuration_Option(t *testing.T) {
+	tm, fn := mockNow()
+	defer fn()
+
+	newClaimer := func(claims *Claims) *Claimer {
+		c, err := NewClaimer(claims, globalProvisionerClaims)
+		if err != nil {
+			t.Fatal(err)
+		}
+		return c
+	}
+	unix := func(d time.Duration) uint64 {
```

Comment:
Why did you decide to use `unix` time for the ssh stuff but the golang Time elsewhere?

Reply:
This is how ssh certificates are defined, and I'm using a library that requires them in that format.